### ISSUE TITLE
Add fprintd and necessary support in gnome

### ISSUE THREAD

**Issue by Jacalz** [clearlinux/distribution#719]
Fedora currently have a stunningly great integration of [fprintd](https://www.freedesktop.org/wiki/Software/fprint/) as the fingerprint reader manager and driver utility that makes it possible to add fingerprints and unlock the system using them. I suggest having it packaged and possibly installed by default along with the necessary support needed. All the things are possible and done in Fedora except for the password needing to be used to change the fingerprint, it was suggested in the arch wiki. Please add more stuff to the list if there was something I missed when using Fedora!

## Integration points:

- [ ] Add the `fprintd` package bundle with `libfprint`
- [ ] Enable unlocking of sudo password once fingerprint is set (it switches over to normal password if not fingerprint is entered, but timeout could be shorter than on Fedora)
- [ ] Force need to use password to change and add password (not done in Fedora yet)
- [ ] Perhaps add documentation on how to set up the fingerprint on Clear Linux

### Gnome specifics:

- [ ] Allow fingerprint unlocking of GDM when logging in to Gnome desktop
- [ ] Wire in support in to the gnome-control-center
- [ ] Unlocking gnome-screensaver using fingerprint

## Additional points of reference:
Fingerprint integration page on Fedora wiki can be found [here](https://fedoraproject.org/wiki/Features/Fingerprint).
The ArchWiki page can be found [here](https://wiki.archlinux.org/index.php/Fprint).

## Some image examples:
![image](https://user-images.githubusercontent.com/25466657/57636041-b13c8600-75a8-11e9-8fbd-124f89849b02.png)

![image](https://user-images.githubusercontent.com/25466657/57636059-ba2d5780-75a8-11e9-8a46-c782ce16e1e5.png)

![image](https://user-images.githubusercontent.com/25466657/57636076-c0bbcf00-75a8-11e9-9b6b-25d87f1a7d23.png)

**Comment by bryteise** [comment]
Hrm @mrkz, @bktan8 do you have any systems around with a fingerprint reader?

**Comment by bktan8** [comment]
@bryteise - Yes, I have a Lenovo X1 Yoga.

**Comment by mrkz** [comment]
Yes,  got a Lenovo T570 to play with, yet I'd like to focus more on other issues on my pile, but really looking forward to get this working on CLR :tada:

**Comment by warnaud** [comment]
Looking forward for it too ! I have and x230 to test if interested, I could try to make a bundle if I have a comprehensive doc on how to make one that specific

**Comment by JamesFrederik** [comment]
Fingerprint support would be very much appreciated.

**Comment by Aqua1ung** [comment]
@fenrus75, any chance we could see fprintd added to CL bundles please? Or is that too much of a security risk?

**Comment by Aqua1ung** [comment]
> I have added the package (and bundle) but I have no way to test this.... so I'll need to depend on y'all to give it the thumbs up or down

Excellent, thank you! I'll do my best to test it and report back!

**Comment by worldofgeese** [comment]
@fenrus75 I am able to enroll but receive "Permission denied" when attempting to use my fingerprint at the GDM login screen.

**Comment by Bear-Essentials** [comment]
Ive added the bundle on my X13 Yoga gen2 and works without issue

**Comment by rlocas** [comment]
To get GDM login to work with the fingerprint scanner, I copied /usr/share/pam.d/gdm-fingerprint to /etc/pam.d/ and changed the line `session  include        system-session` to `session  include        common-session`. This is how it is in gdm-password.
